test/js/simple-length.js

```diff
@@ -90,4 +90,24 @@ suite('SimpleLength', function() {
     assert.strictEqual(lengthAddition.value, 5);
   });
 
+  test('convertToCalcLength method returns a CalcLength object with a single length value type when called from a SimpleLength object', function() {
+    var simpleLength = new SimpleLength(10, 'em');
+    var convertedCalcLength = simpleLength._convertToCalcLength();
+    var testCalcLength = new CalcLength({em: 10})
+    assert.isTrue(convertedCalcLength.equals(testCalcLength));
+  });
+
+  test('Test equals method of Simple length such that it will return true if two simple lengths are the same', function() {
+  var simpleLength_1 = new SimpleLength(10, 'em');
+  var simpleLength_2 = new SimpleLength(10, 'em');
+  
+  assert.isTrue(simpleLength_1.equals(simpleLength_2));
+  });
+
+  test('Equals method should return false if a CalcLength is compared to a simple length even if they have the same value', function() {
```

Comment:
How about "Equals method should return false if any other kind of object is passed in"

then you can also try

assert.isFalse(simpleLength.equals({}));
assert.isFalse(simpleLength.equals(5));